tests/unit/main.c

```diff
@@ -545,6 +566,13 @@ ctest_usage(const char *progname, int program_is_unit_test)
    } else {
       printf("[ --list | [ --<config options> ]* <test-case-name> ]\n");
    }
+   printf(
+      "\nUse env-var VERBOSE=<n> to control verbosity of output from tests.\n");
+   printf("  %d - Or, unset, turns off all output from tests. (Default).\n",
+          MSG_LEVEL_SILENT);
+   printf("  %d - Only error messages are printed.\n", MSG_LEVEL_ERRORS);
```

Comment:
This isn't correct right now, since `MSG_LEVEL_ERRORS` is 3, but the `ctest_get_verbosity()` function silently treats 3 the same as 0.  I suggest fixing `ctest_get_verbosity()`.

Reply:
Not sure how to fix this in `ctest_get_verbosity` for  handling 3. Only `VERBOSE=1` is being allowed now and the `1` is mapped to `MSG_LEVEL_ERRORS`.

I agree the help message reporting 3 is incorrect. I think I should just change that to hard-coded 1 as that's how env-var `VERBOSE=1` is operating.

Let's talk ...

Reply:
@rosenhouse : I fixed the `--help` output to now report 1 for printing errors. The output looks like:
```
Use env-var VERBOSE=<n> to control verbosity of output from tests.
  0 - Or, unset, turns off all output from tests. (Default).
  1 - Only error messages are printed.
  7 - All messages, including debugging output is printed.
```

I think that should be sufficient.

I did think about your comment of `suggest fixing ctest_get_verbosity().` I thought of amending this code-fragment in that function:

```
 428    if (STRING_EQUALS_LITERAL(val, "1")) {
 429       return MSG_LEVEL_ERRORS;
 430    }
```

to also allow 3; i.e. change L428 to : `if (STRING_EQUALS_LITERAL(val, "1") || STRING_EQUALS_LITERAL(val, "3"))`

That would take us a bit closer to the eventual msg-level of `error == 3`.

But I figured that as eventually we plan to remove this `VERBOSE=` env-var support in lieu of a proper `--verbose*` cmd-line flag, I felt it's not that critical to "improve" the handling at this time.

Let me know if you disagree and still wish to see something else done here.